core/src/main/java/io/grpc/Status.java

```diff
@@ -377,8 +376,7 @@ public static Status fromCode(Code code) {
   /**
    * Key to bind status message to trailing metadata.
    */
-  @Internal
-  public static final Metadata.Key<String> MESSAGE_KEY =
+  static final Metadata.Key<String> MESSAGE_KEY =
```

Comment:
It looks like this and `StatusMessageMarshaller` could be moved to `io.grpc.internal`.